### ISSUE TITLE
cannot install

### ISSUE THREAD

**Issue by PAOK1926** [evolutionscript/HelpDeskZ-1.0#55]
i have created the user and the database,
i go through the installation process, but when i submit the sql details, i see an empty page.
config.php is zero sized, tables, etc are not created in the database...

can someone help me  ?

**Comment by matthewbacon** [comment]
Same here. I suspect you are running PHP 7.x like me. I think they need to update the code to be 7.x compliant.

**Comment by matthewbacon** [comment]
If you take a look at your log file you'll probably see something like this.

[04-Sep-2018 20:31:27 Europe/London] PHP Deprecated:  Methods with the same name as their class will not be constructors in a future version of PHP; Registry has a deprecated constructor in /home/username/support.domain.com/includes/classes/classRegistry.php on line 9

[04-Sep-2018 20:31:27 Europe/London] PHP Deprecated:  Methods with the same name as their class will not be constructors in a future version of PHP; Mailer has a deprecated constructor in /home/username/support.domain.com/includes/classes/classMailer.php on line 10

**Comment by cassiohuber** [comment]
Yes, it is the PHP version, use version 5.6 to do the installation

**Comment by OMEGAYALFA** [comment]
Any solution to work with the version of php 7 ...?

**Comment by judeapana** [comment]
I tried installing it but faced the same issue.
Current Version of php8.0 
Im running it with nginx
